index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Project</title>
+    <meta name="viewport" content="width=device-width">
+    <link href="style.css" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+<header class="main">
+    <h1>Reliable, efficient delivery</h1>
+    <h1><strong>Powered by Technology</strong></h1>
```

Comment:
This is much more concise than my solution haha!  I did some unnecessary css styling for the header. Your code is super neat :)